docs/reference/content/getting-started/quick-tour-primer.md

```diff
@@ -0,0 +1,95 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Quick Tour Primer"
+[menu.main]
+  parent = "Getting Started"
+  identifier = "Primer"
+  weight = 20
+  pre = "<i class='fa'></i>"
++++
+
+# Quick Tour Primer
```

Comment:
If you're just going through the reference docs from start to finish, this page doesn't make sense as is.  It's not clear at first that the Quick Tour Primer is a primer for the Quick Tour, as why would a quick tour need a primer (that doesn't seem quick at all!).  

I think it could be solved by a bit of introductory text at the start of this page explaining how it relates to the quick tour, as well as some additional text on the Getting Started page explaining the purpose of each of the four pages listed there.